src/patternfly/components/CodeEditor/code-editor.scss

```diff
@@ -86,6 +90,13 @@
   }
 }
 
+.pf-c-code-editor__header-main {
+  flex-grow: 1;
+  align-self: center;
```

Comment:
I'm not sure that I would expect this. By default it would do something like this if the tab text font were bigger (just as an example)
<img width="854" alt="Screen Shot 2022-02-08 at 6 48 45 PM" src="https://user-images.githubusercontent.com/35148959/153100918-d38960f4-6ced-471c-8e06-d755541ef58f.png">

I think I would expect it to be `flex-end` like the other elements, and if the children of that element need a different alignment, that could be done with a layout.

I realize it will make the example look like this, but I think that's ok... wdyt @mcarrano?

<img width="852" alt="Screen Shot 2022-02-08 at 6 52 51 PM" src="https://user-images.githubusercontent.com/35148959/153101107-b64822f4-756b-4eed-b6a3-513055c65c17.png">